index.ts

```diff
@@ -8,7 +28,8 @@ let beforeEach = global.beforeEach;
 let afterAll = global.after;
 let afterEach = global.afterEach;
 
-let nodeSymbol = global.Symbol || (key => "__mts_" + key);
+//let nodeSymbol = global.Symbol || (key => "__mts_" + key);
+let nodeSymbol = (key => "__mts_" + key);
```

Comment:
I really liked the way it worked with symbols, but since symbol seem to cause problems on some node versions, perhaps it really is best to drop the idea.